src/pages/unexpected-error/home/home.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+
+import globalStyles from 'assets/stylesheets/global-styles.module.scss';
+import { AppLink } from 'routes/app-link';
+import { RouteName } from 'routes/routes';
+import { Button } from 'common/button';
+import { Avatar } from 'common/avatar';
+import { Modal } from 'common/modal';
+import { ModalSizes } from 'common/modal/modal';
+import { TextField, TextFieldStatus } from 'common/text-field';
+import { TextArea } from 'common/text-area';
+import { mailFormat } from 'helpers/utils';
+import styles from './home.module.scss';
+import { ReactComponent as MailSVG } from '../../assets/icons/mail.svg';
+import { ReactComponent as CloseSVG } from '../../assets/icons/close.svg';
+
+const Home = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const closeModal: () => void = () => {
+    setOpenModal(false);
+  };
+  const [inputState, setInputState] = useState('');
+  const [textAreaState, settextAreaState] = useState('random text');
+  const [helperIState, setHelperIState] = useState('');
+  const [helperTAState, setHelperTAState] = useState('');
+
+  const charactersLimit = 35;
+
+  useEffect(() => {
+    if (!inputState.match(mailFormat) && inputState !== '') {
+      setHelperIState('Not a valid email format');
+    } else {
+      setHelperIState('');
+    }
+  }, [inputState]);
+
+  useEffect(() => {
+    if (textAreaState?.length < 10 && textAreaState?.length > 0) {
+      setHelperTAState('At least 10 characters needed');
+    } else if (textAreaState?.length === charactersLimit) {
+      setHelperTAState('Character limit reached');
+    } else setHelperTAState('');
+  }, [textAreaState]);
+
+  const handleIChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInputState(e.target.value);
+  };
+  const handletextAreaChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    settextAreaState(e.target.value);
+  };
+  const handleClear = () => {
+    setInputState('');
+  };
+  return (
+    <div className={globalStyles.genericContainer}>
+      <h1 className={styles.title}>
+        <Avatar size="xl" />
+        <hr />
+        Welcome! This is the HomePage.
+        <AppLink routeName={RouteName.About}>About</AppLink>
+        <Button onClick={() => setOpenModal(true)}>Hello World</Button>
+      </h1>
+      <div className={styles.formContainer}>
+        <form className={styles.form}>
+          <h3 style={{ backgroundColor: 'lightgray', marginBottom: '48px' }}>Text Field</h3>
```

Comment:
It's okey this text?

Reply:
@agustinasalmanton This is from the template, I didn't created it